### ISSUE TITLE
expose multi_label_per_box for yolonas during inference - (PPYoloEPostPredictionCallback)

### ISSUE THREAD

**Issue by DanTaranis** [Deci-AI/super-gradients#1736]
### 🚀 Feature Request

PPYoloEPostPredictionCallback has an option to predict 1 label per box - which is more consistent with my usecase.
However during infer - I don't see an option to have the model output 1 output per bbox.

### Proposed Solution (Optional)

allow to specify a inference_params similar to train_params?
is it maybe possible through here: ```model.set_dataset_processing_params```

**Comment by BloodAxe** [comment]
Indeed this would be a good feature to have. We will add it in a next releases.

**Comment by BloodAxe** [comment]
https://github.com/Deci-AI/super-gradients/pull/1738

**Comment by BloodAxe** [comment]
This will become available in 3.6 release

**Comment by BloodAxe** [comment]
Can you show us the code example that demonstrate the issue you are facing?

**Comment by DanTaranis** [comment]
```
    model = models.get(MODEL_ARCH,
                checkpoint_path=<././././>/ckpt_best.pth",
                num_classes=NUM_CLASSES)
    with torch.no_grad():
                    preds_ = model.predict(file_paths, conf=.1, batch_size=8, iou=.5
                                                          multi_label_per_box=False,  max_predictions=50,nms_top_k=300, nms_threshold=.7)
```

**Comment by DanTaranis** [comment]
please check out the comments here as well:
https://github.com/Deci-AI/super-gradients/pull/1738
